### PR TITLE
J oakes new entity event

### PR DESCRIPTION
**Pull request by joakes0797**

### CODE REVIEW

File: PickUpEventsApp.Data/Model/Event.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        public User Owner { get; set; }
+
+        public string Name { get; set; }
+
+        public string Description { get; set; }
+
+        public IEnumerable<User> Users { get; set; } 
+
+        // This is a circular reference to this class.
+        public IEnumerable<Event> Events { get; set; }
```
Comment (uhyeay2) [inline PickUpEventsApp.Data/Model/Event.cs]: I am curious on what the circular reference means?

To make sure I understand properly, will Events have a collection of Events that belong to them?

Comment (joakes0797) [reply PickUpEventsApp.Data/Model/Event.cs]: Discussed in meeting, property removed.

**Comment by joakes0797** [comment]
This PR contains the creation of the Event and Group entities.
I consolidated the other entities and my additions into this PR and moved everything into the PickUpEventsApp.Data Model folder.
There are now 4 entities: User, Location, Group, and Event.

**Review by TheMrChaptastic — APPROVED** [review]
Nice job!

**Pull request merged**